### PR TITLE
Added footer

### PR DESCRIPTION
**Pull request by Himanshu-370**
@shreya024 

Added footer to the main App.js
image

Created a new folder Footer with new files footer.jsx and footer.css
image

final preview
image

### CODE REVIEW

File: client/memories/src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <div className="footer-items">
+          <p>Terms of Service</p>
+          <p>Privacy Policy</p>
+          <p>Cookie Policy</p>
+          <p>Accessibility</p>
+          <p>More</p>
+        </div>
+      </div>
+      <div className="footer-bottom">
+        <p>© 2022 Memories App</p>
```
Comment (soumali28) [inline client/memories/src/components/Footer/Footer.jsx:20]: Could you add a dynamic date instead of hardcoding it?

Comment (Himanshu-370) [reply client/memories/src/components/Footer/Footer.jsx:20]: Sure!

**Review by soumali28 — CHANGES_REQUESTED** [review]
Could you add a dynamic date in the footer instead of hard coding it.

**Comment by Himanshu-370** [comment]
Can you see the change?

**Review by soumali28 — APPROVED** [review]

**Comment by soumali28** [comment]
@Himanshu-370 Yep

**Pull request merged**